### DISCUSSION TITLE
How to "Send and Download" a http Get request?

### DISCUSSION THREAD

**Discussion by kengjungwa** [usebruno/bruno#325] [Q&A]
Hi,

I'd like to ask if there is a way to use Bruno Desktop to **send a HTTP Get request** and **download the response as a file i.e. xlsx or PDF** similar to Postman (in the following snapshot).
![image](https://github.com/usebruno/bruno/assets/1743535/a0a9cf75-0a2e-4013-a621-e049d607f65e)

In Bruno CLI doc (https://docs.usebruno.com/cli/overview.html), I found the option `--output` in the doc. Not sure if this option is the right one and, if so, how it can be used in the Bruno Desktop.

Thanks a lot for such a great tool!
Best,
Keng

**Comment by emirozmen07** [comment]
After sending the request you can download with the little icon above the response 
![image](https://github.com/usebruno/bruno/assets/144101618/e1516da5-2ae4-4a28-9569-e15bdf9d0439)

**Reply by merc1er** [reply]
I'm not seeing that icon on the latest version (v3.0.2):

<img width="577" height="136" alt="image" src="https://github.com/user-attachments/assets/36c4e072-540f-4cad-853e-310fc6e35c07" />

**Reply by emirozmen07** [reply]
if you click to three dots it's there
<img width="919" height="210" alt="image" src="https://github.com/user-attachments/assets/0516a4c4-5c2a-443e-b40f-908176f27a0c" />

**Reply by merc1er** [reply]
There are no 3 dots as shown in my screenshot. It looks like a fix has been made, but it's not released yet: https://github.com/usebruno/bruno/issues/6910

Edit: this only affects GraphQL queries.
